dist/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
Je mag de dist folder in de gitignore zetten, je hebt dit alleen nodig voor productie.